sdk/haskell/dstar.cabal

```diff
@@ -0,0 +1,121 @@
+cabal-version:      3.12
+name:               dstar
+version:            0.2.0.0
+synopsis:           Interface for Server Sent Events using the DataStar framework
+description:        See https://data-star.dev/ for full information
+                    about the wonderful new (2024) backend framework,
+                    almost totally powered by the frontend.
+license:            NONE
```

Comment:
The license should match Datastar’s: MIT License
https://github.com/starfederation/datastar/blob/develop/LICENSE